sdk/communication/AzureCommunicationChat/Source/NotificationUtils/CryptoUtils.swift

```diff
@@ -142,15 +142,37 @@ func decryptPushNotificationPayload(cipherText: [UInt8], iv: [UInt8], cryptoKey:
     return String(decoding: cryptData, as: UTF8.self)
 }
 
-public struct EncryptionKeyPair {
-    let firstKey: String
-    let secondKey: String
-
-    public init(
-        firstKey: String,
-        secondKey: String
-    ) {
-        self.firstKey = firstKey
-        self.secondKey = secondKey
+internal func generateEncryptionKey() -> String {
+    return SymmetricKey(size: .init(bitCount: 512)).serialize()
+}
+
+// Use the SymmetricKey class in CryptoKit framework to create encryption keys
+extension SymmetricKey {
+    /// Serializes a `SymmetricKey` to a Base64-encoded `String`.
+    func serialize() -> String {
+        return withUnsafeBytes { body in
+            Data(body).base64EncodedString()
+        }
```

Comment:
This seems, well, unsafe. What is the status of consulting the Crypto Board for their review?

Reply:
In order to convert a Symmetric key into a base64EncodedString, we will need to access the underlying data of the Symmetric key first.
Based on Apple Official doc, [func withUnsafeBytes<R> -> R] is the only way to achieve it. From my understanding, this is how Apple named it.
https://developer.apple.com/documentation/cryptokit/symmetrickey

And we've confirmed with the PNH team and got a response that they had this feature reviewed by Crypto Board before.